db/migrate/20180524190818_destroy_non_existing_git_hub_users.rb

```diff
@@ -0,0 +1,12 @@
+class DestroyNonExistingGitHubUsers < ActiveRecord::Migration[5.1]
+  def up
+    unless ENV['RAILS_ENV'] == 'development'
+      User.find_each do |user|
+        if user.github_id == nil
+          puts "removing user #{user.id} from database"
+          user.destroy
```

Comment:
This may not matter much, depending on the number of records we have, but I think this version leads to better performance:

```ruby
User.where(github_id: nil).destroy_all.each do |user|
  puts "removed user #{user.id} from database"
end
```

`destroy_all` returns the records that were removed, so we can use those to print stuff.

PS: did a quick one-run test and for 5000 users this method ran in ~23 seconds while the inline `user.destroy` method ran in ~41 seconds. That being said, this probably doesn't matter at all since we don't have a lot of records being destroyed! 😅